main.cpp

```diff
@@ -332,7 +201,35 @@ Thing 1) Foley Studio
     2) edit sfx and prepare to mix
     3) upload sfx to cloud
  */
-
+struct FoleyStudio
+{
+    FoleyStudio();
+    unsigned int numberOfPits;
+    unsigned int numberOfShoes;
+    unsigned int numberOfCloths;
+    unsigned int sfxPerDay;
+    unsigned int sfxEditPerDay;
+
+    struct Sneaker
+    {
+      Sneaker();
+      bool isThisNew = false;
+      std::string brand = "Adidas";
+      std::string model = "Superstar";
+      double shoeSize = 42.5;
+      bool isThisLaceUp = true;
+      
+      void getWearSneaker(int legs = 2, std::string gender = "Female");
+      float setupMicVolume = 32.f;
+      void performFeetSfx(double tempo = 100.20);
+    };
```

Comment:
2 things need revising here:
- You have 6 variable declarations and 2 function declarations.  The instructions specified that 5 variables and 3 functions need to be declared.
- The code inside the `{ }` needs to be indented 4 spaces relative to the `{`
Please correct the issues
- [ ] revise